src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -73,10 +79,19 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField("User ID", userForInfo.getId(), true);
         builder.addField("Mutual Servers", String.valueOf(userForInfo.getMutualGuilds().size()), true);
 
-
+        if (member != null) {
+            PermissionsManager permissionsManager = new PermissionsManager();
+            long ids = Long.parseLong(userForInfo.getId());
+            SecurityLevel userSecurityLevel = permissionsManager.getUserSecurityLevel(ids);
+            if (userSecurityLevel != null) {
+                builder.addField("Official Roles", userSecurityLevel.name(), true);
```

Comment:
Can we send this to lowercase? It doesn't look like it fits being in all caps, when everything else is formatted.

Reply:
Will do

Reply:
nagh we want it properly capitalized. not just all lower case or upper.

Reply:
That's what I will do

Reply:
That's what I meant oops